niceday-api/api/openapi.yaml

```diff
@@ -193,3 +206,76 @@ components:
           description: Whether the tracker should be enabled
           example: true
 
+    TrackerReminder:
+      description: Reminder for a user tracker
+      required:
+      - userId
+      - recurringSchedule
+      type: object
+      properties:
+        userId:
+          type: string
+          description: ID of the user for which the reminder has to be set
+          example: "38527"
+        recurringSchedule:
+          $ref: '#/components/schemas/RecurringSchedule'
+
+    RecurringSchedule:
+      description: Recurring expressions object
+      required:
+      - title
+      - schedule_type
+      - recurring_expression
+      type: object
+      properties:
+        title:
+          type: string
+          description: Titile of the schedule
+          example: 'Smoking entry reminder'
+        schedule_type:
+          type: string
+          description: Type of schedule to set
+          example: 'tracker_smoking'
+        recurring_expression:
+          $ref: '#/components/schemas/RecurringExpression'
+
+    RecurringExpression:
+      description: Recurring expression object
+      required:
+      - margin
+      - reminder_enabled
+      - reminder_margin
+      - rrule
+      type: object
+      properties:
+        margin:
+          $ref: '#/components/schemas/Margin'
+        reminder_enabled:
+          type: boolean
+          description: Flag for enabling reminder
+          example: true
+        reminder_margin:
+          type: array
+          items:
+            $ref: '#/components/schemas/Margin'
+        rrule:
+          type: string
+          description: recursive rule
```

Comment:
This needs some more documentation, or more examples. Or reference to the standard object documentation.